DbProvider/PlayerMover.cs

```diff
@@ -46,17 +72,22 @@ public void MoveTowards(Vector3 destination)
             if (Core.IsOutOfGame)
                 return;
 
-            if (DateTime.UtcNow < SleepUntilTime)
+            if (DateTime.UtcNow < _sleepUntilTime)
                 return;
 
-            if (!ZetaDia.IsInGame || ZetaDia.Me == null || !ZetaDia.Me.IsValid || ZetaDia.Me.IsDead || ZetaDia.Globals.IsLoadingWorld)
+            if (!ZetaDia.IsInGame ||
+                ZetaDia.Me == null ||
+                !ZetaDia.Me.IsValid ||
+                ZetaDia.Me.IsDead ||
+                ZetaDia.Globals.IsLoadingWorld)
                 return;
 
-            if (UiSafetyCheck())
+            if (IsUIBlockingMovement)
                 return;
 
             var power = TrinityCombat.Routines.Current.GetMovementPower(destination);
-            if (power == null || power.SNOPower == SNOPower.None)
+            if (power == null ||
+                power.SNOPower == SNOPower.None)
                 power = new TrinityPower(SNOPower.Walk, 3f, destination);

```

Comment:
Add curly brackets when the statement is multi-line. Although the statement can be one liner here